src/components/profile/header/header.profile.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import "./header.profile.scss";
+import mainProfilePicture from "./profile1.jpeg";
+import searchIcon from "./mag.png";
+import addFriendIcon from "./add.png";
+
+type HeaderProps = {
+  onSearchChange: React.ChangeEventHandler<HTMLInputElement>;
+};
+
+function Header({ onSearchChange }: HeaderProps) {
+  let [searchToggle, setSearchToggle] = useState(false);
+  return (
+    <>
+      <div className="Header">
+        <h1>친구</h1>
+        {searchToggle === true ? (
+          <input placeholder="검색" onChange={onSearchChange} />
+        ) : null}
```

Comment:
```suggestion
        {searchToggle && (
          <input placeholder="검색" onChange={onSearchChange} />)}
```
이 부분도 이렇게 써 볼 수 있을 것 같아요